autoload/lightline/bufferline.vim

```diff
@@ -127,11 +129,54 @@ function! s:delete_nth_buffer(n)
   endif
 endfunction
 
+function! s:get_buffer_paths(buffers)
+  if (s:smart_path)
```

Comment:
To reduce indentation please negate the if expression to turn the check into a guard and remove the else:
```viml
  if (!s:smart_path)
    return map(a:buffers, 'bufname(v:val)')
  endif

  ...
```